content/blog/2024-11-18-announcing-95-stable.md

```diff
@@ -0,0 +1,71 @@
+---
+title: "General Availability of AlmaLinux 9.5 Stable!"
+type: blog
+author:
+ name: "Eduard Abdullin"
+ bio: "The Release and Automation Engineer."
+ image: /users/eduard-abdullin.jpg
+date: '2024-11-18'
+images:
+  - /blog-images/2024/2024-11-18-almalinux-9-5-stable.png
+post:
+    title: "General Availability of AlmaLinux 9.5 Stable!"
+    image: /blog-images/2024/2024-11-18-almalinux-9-5-stable.png
+
+---
+
+# AlmaLinux 9.5 Stable Now Available 
+
+Hello Community! The AlmaLinux OS Foundation is announcing the general availability of [AlmaLinux OS 9.5](https://mirrors.almalinux.org/isos.html) codenamed "Teal Serval"!
+
+Installation ISOs are available on the mirrors now for all 4 architectures:
+* [Intel/AMD (x86_64)](https://mirrors.almalinux.org/isos/x86_64/9.5.html)
+* [ARM64 (aarch64)](https://mirrors.almalinux.org/isos/aarch64/9.5.html)
+* [IBM PowerPC (ppc64le)](https://mirrors.almalinux.org/isos/ppc64le/9.5.html)
+* [IBM Z (s390x)](https://mirrors.almalinux.org/isos/s390x/9.5.html)
+
+Torrents are available as well at:
+* [Intel/AMD (x86_64)](https://repo.almalinux.org/almalinux/9.5/isos/x86_64/AlmaLinux-9.5-x86_64.torrent)
+* [ARM64 (aarch64)](https://repo.almalinux.org/almalinux/9.5/isos/aarch64/AlmaLinux-9.5-aarch64.torrent)
+* [IBM PowerPC (ppc64le)](https://repo.almalinux.org/almalinux/9.5/isos/ppc64le/AlmaLinux-9.5-ppc64le.torrent)
+* [IBM Z (s390x)](https://repo.almalinux.org/almalinux/9.5/isos/s390x/AlmaLinux-9.5-s390x.torrent)
+
+## ISOs, Live Images, Cloud and Containers
+
+AlmaLinux also offers a variety of Cloud, Container and Live Images. The builds for these get kicked off as soon as the public repository is ready. 
+
+**The following images are expected to be available shortly.** 
```

Comment:
Is it feasible to wait until these are live before publishing? Or are these images not at all ready, which will hold up the release?

Reply:
Talked to Elkhan who helped clarify the issue with releasing these at the same time.

Reply:
Putting the answer for the backlog purpose to the community.

We have plans to build cloud and other images using the development versions of ISO images and repositories to be able to publish at the same time with the stable ISO images.

We have not implemented this plan yet because having the same day images may result with the errors with the package manager, the `dnf` due the out-of-sync nearest mirrors to the users. Currently, We wait until majority of the mirrors sync the new versions of AlmaLinux OS.